library/src/compute/calc.rs

```diff
@@ -131,6 +131,43 @@ pub fn pow(
     result.value()
 }
 
+/// Raise a value to some exponent of e.
+///
+/// ## Example { #example }
+/// ```example
+/// #calc.exp(3)
+/// ```
+///
+/// Display: Power
+/// Category: calculate
+/// Returns: integer or float
```

Comment:
```suggestion
/// Returns: float
```
In your code, it always returns a `float`.